CompanionApp/Companion/Companion/MainMenuViewModel.cs

```diff
@@ -30,6 +30,14 @@ public MainMenuViewModel(INavigation navigation, Action<Page> openPageAsRoot)
             {
                 MainMenuSelectedItem = firstEntry;
             }
+
+            _navigation.PushModalAsync(new NavigationPage(new Questionnaire()));
+
+            //Check for Questionnaire:
+            //if (DateTime.Now.CompareTo(App.QuestionnaireLastCompleted.AddMonths(1)) >= 0)
+            //{
+            //    _navigation.PushAsync(new Questionnaire());
+            //}
```

Comment:
It may be worthwhile to make a card to turn this back on and test it. Or to pop it open if a flag is set.